packages/debug/src/browser/console/debug-console-session.ts

```diff
@@ -23,10 +23,11 @@ import URI from '@theia/core/lib/common/uri';
 import { ExpressionContainer, ExpressionItem } from './debug-console-items';
 import { Severity } from '@theia/core/lib/common/severity';
 import { injectable, postConstruct } from '@theia/core/shared/inversify';
+import { ConsoleSessionManager } from '@theia/console/src/browser/console-session-manager';
```

Comment:
@martin-fleck-at importing from `src/` will cause the build to fail:
```suggestion
import { ConsoleSessionManager } from '@theia/console/lib/browser/console-session-manager';
```

Reply:
Ah, quick fix import from vscode got me again! Thanks!